subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -562,9 +572,17 @@ static void isr_window(void *param)
 {
 	uint32_t ticks_at_start;
 	uint32_t remainder_us;
+	struct lll_scan *lll;
 
 	isr_common_done(param);
 
+	lll = param;
+	if (++lll->chan == 3U) {
+		lll->chan = 0U;
+	}
```

Comment:
`3U` seems like a magic number in this context. Hard to tell what's going on here. Consider adding a comment about why comparing to 3, and why it's `++lll->chan` rather than `lll->chan`.